_content/symptoms-and-testing/kind-of-test-used-to-diagnose-covid19.md

```diff
@@ -1,23 +1,16 @@
 ---
 category: symptoms-and-testing
-date: April 21, 2020
+date: April 22, 2020
 excerpt: Symptoms and testing
 layout: post
 promoted: false
 sources:
 - agency: cdc
   url: https://www.cdc.gov/coronavirus/2019-ncov/faq.html#Symptoms-&-Testing
-- agency: fda
-  url: https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/coronavirus-disease-2019-covid-19-frequently-asked-questions
-title: What kind of test is being used to diagnose if I have COVID-19?
```

Comment:
It looks like the title is missing.

Reply:
Fixed; see preveiw: https://cg-dae9433e-23a0-46d7-bafb-dc79d2d3756c.app.cloud.gov/preview/18f/cv_faq/cdc-update-4-23/symptoms-and-testing/